### ISSUE TITLE
Control+` hot key doesn't work

### ISSUE THREAD

**Issue by ashevtsov-mt** [p0deje/Maccy#19]
I'm trying to set `` Control+` `` as my hot key but Maccy wouldn't show up when I hit the combination (after restarting the app, of course).

Here is what I tried so far:
```
defaults write org.p0deje.Maccy hotKey control+\`
defaults write org.p0deje.Maccy hotKey 'control+`'
defaults write org.p0deje.Maccy hotKey control+\\\`
```

I used [Perf Setter](http://www.nightproductions.net/prefsetter.html) to confirm that the setting is indeed `` control+` ``
Any help would be much appreciated.

**Comment by p0deje** [comment]
As a workaround, please use the following:

```
defaults write org.p0deje.Maccy hotKey control+grave
```

I'll find a better solution.

**Comment by ashevtsov-mt** [comment]
It works. Thank you!
I wouldn't call it a bug, though. Just worth mentioning in `README.md`

**Comment by p0deje** [comment]
This is fixed in bee1440ef1cc10f9b1af12cfe429030d9aace4f8 and will be available in the next release.
